pkgs/top-level/all-packages.nix

```diff
@@ -12877,6 +12877,8 @@ with pkgs;
 
   sanjuuni = callPackage ../tools/graphics/sanjuuni { };
 
+  sarif-tools = with python3Packages; toPythonApplication sarif-tools;
```

Comment:
write the code below into `pkgs/by-name/sa/sarif-tools/package.nix`:

```nix
{ python3Packages }:

with python3Packages; 
toPythonApplication sarif-tools;
```

and delete the lines in all-packages.